onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_api_helper.h

```diff
@@ -54,12 +54,16 @@ constexpr const char* const kNnapiCpuDeviceName = "nnapi-reference";
  */
 int32_t GetNNAPIEffectiveFeatureLevel(const NnApi& nnapi_handle, gsl::span<const DeviceWrapper> device_handles);
 
+// The value 3 is somewhat arbitrary. It's accounting for one each of ANEURALNETWORKS_DEVICE_ACCELERATOR,
+// ANEURALNETWORKS_DEVICE_CPU, and ANEURALNETWORKS_DEVICE_GPU.
+constexpr size_t kDefaultDeviceWrapperInlinedVectorSize = 3;
```

Comment:
was hitting static_assert in MSVC about InlinedVector size being too big, so explicitly specifying a count

Reply:
Does this get rounded up to a power or 2 so might as well be 4?

Reply:
I don't think it does - as far as I can tell N gets propagated to where the inlined storage array is declared:
https://github.com/abseil/abseil-cpp/blob/8c0b94e793a66495e0b1f34a5eb26bd7dc672db0/absl/container/internal/inlined_vector.h#L467-L469

Reply:
not specifying a size anymore, made a change in the inlined vector header so the assertion no longer fails